README.md

```diff
@@ -51,7 +51,25 @@ Once you have completed the steps in the Configuration section above, you are re
 
 Testing the Payroll Income flow requires modifying the steps above. Instead of calling "Create Item [Sandbox Only]", go to the Income folder and call "Create User Token", followed by "Initialize User Token for Payroll Income [Sandbox only]", and then finally "Retrieve Payroll Income".
 
-The Bank Income flow cannot be tested using the instructions above. To test Bank Income, use the instructions below for making Postman calls with real data; but you may optionally use Sandbox instead of Production or Development. 
+The Bank Income flow cannot be tested using the Quickstart instructions above. To test Bank Income, use the instructions below for making Postman calls with real data; but you may optionally use Sandbox instead of Production or Development. 
+
+### Transfer Quickstart notes
+
+Using the Transfer endpoints requires you to be enrolled in the Transfer beta. Contact your Plaid Account manager or Plaid Sales to join the beta.
+
+To call Transfer endpoints, you will need either a payment profile token, or an both an access token and an account id. To obtain a payment profile token, call Transfer -> Create Payment Profile. To obtain an access token and account id, follow the Quickstart steps above, then call Items -> Item Management -> Retrieve an Item's accounts. 
+
+Once you have a payment profile token or an access token/account id pair, start by calling Transfer -> Authorize a transfer, then call Transfer -> Initiate a Transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using Payment Profiles, delete the `account_id` and `access_token` fields before making the request.
```

Comment:
```suggestion
Once you have a payment profile token or an access token / account id pair, start by calling Transfer -> Authorize a transfer, then call Transfer -> Initiate a transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using Payment Profiles, delete the `account_id` and `access_token` fields before making the request.
```

(Also, "you are choosing" sounded weird, but wasn't sure if "you selected above" was any better.